queries/beancount/highlights.scm

```diff
@@ -0,0 +1,26 @@
+
+(date) @field
+(txn) @annotation
+
```

Comment:
@vigoux @steelsojka should we use `@anotation` and `@attribute` the same way as the Atom queries? (Source code annotations/attributes and `@attribute` for html/xml attributes)

Reply:
changed it to attribute